setup

```diff
@@ -4,13 +4,35 @@ set -e
 # Just set this to zero if you'd rather keep the build directory around.
 DO_CLEANUP=1
 
+# Set this to 1 to accept default prompts
+ACCEPT_DEFAULTS=0
+
 # Attempts to determine the number of cores in the CPU
 # Source: https://gist.github.com/jj1bdx/5746298
 CPUS=$(getconf _NPROCESSORS_ONLN 2>/dev/null)
 [ -z "$CPUS" ] && CPUS=$(getconf NPROCESSORS_ONLN)
 [ -z "$CPUS" ] && CPUS=$(ksh93 -c 'getconf NPROCESSORS_ONLN')
 [ -z "$CPUS" ] && CPUS=1
 
+while getopts 'y' flag; do
```

Comment:
Perhaps let this be `while getopts 'yh' flag` so `-h` is recognized.

Reply:
good catch!  I pushed a commit to fix it

Reply:
Thanks!!